mtools/mloginfo/sections/connection_section.py

```diff
@@ -19,6 +20,7 @@ def __init__(self, mloginfo):
 
         # add --restarts flag to argparser
```

Comment:
Not directly related to this change, but the comment is incorrect so you could fix if you are adding extra args

Reply:
done